corehq/apps/accounting/models.py

```diff
@@ -1332,7 +1332,8 @@ def set_billing_account_entry_point(self):
     def _get_plan_by_subscriber(cls, subscriber):
         active_subscriptions = cls.objects\
             .filter(subscriber=subscriber, is_active=True)\
-            .order_by('-date_created')[:2]
+            .order_by('-date_created')[:2]\
+            .select_related('plan_version__role')
```

Comment:
This eliminates two queries, and instead uses INNER JOINs to fetch the related objects. Maybe I should trace the call sites of this function to see if a parameter to make this optional would make sense (for example, if some call sites are not accessing `plan_version` and/or `plan_version.role`).